Importing all the packages

In [3]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns

Data Collection:

In [5]:
data_test = '/Users/sangth/Desktop/Capstone-2/Dataset_santander-value-prediction-challenge/test.csv'
data_train = '/Users/sangth/Desktop/Capstone-2/Dataset_santander-value-prediction-challenge/train.csv'

Read the dataset

In [7]:
df_test = pd.read_csv(data_test)
df_train = pd.read_csv(data_train)

Explore the dataset

In [14]:
print('Shape of the test csv dataset :', df_test.shape)
print('Shape of the train csv dataset :', df_train.shape)

print('Shape of the test csv dataset :', df_test)
print('Shape of the train csv dataset :', df_train)

print("\nSummary statistics:\n", df_test.describe())


Shape of the test csv dataset : (49342, 4992)
Shape of the train csv dataset : (4459, 4993)
Shape of the test csv dataset :               ID  48df886f9  0deb4b6a8  34b15f335  a8cb14b00  2f0771a37  \
0      000137c73        0.0        0.0        0.0        0.0        0.0   
1      00021489f        0.0        0.0        0.0        0.0        0.0   
2      0004d7953        0.0        0.0        0.0        0.0        0.0   
3      00056a333        0.0        0.0        0.0        0.0        0.0   
4      00056d8eb        0.0        0.0        0.0        0.0        0.0   
...          ...        ...        ...        ...        ...        ...   
49337  fff73b677        0.0        0.0        0.0        0.0        0.0   
49338  fff7b5923        0.0        0.0        0.0        0.0        0.0   
49339  fff7c698f        0.0        0.0        0.0        0.0        0.0   
49340  fff8dba89        0.0        0.0        0.0        0.0        0.0   
49341  fffbe2f6f        0.0        0.0        0.0  

##### Shape of the test csv dataset rows & columns : (49342, 4992)
##### Shape of the train csv dataset rows & columns : (4459, 4993)

In [15]:
 df1 = 49342 / 4459
 print(df1)

11.065709800403678


In [16]:
df_test.head()

ID  48df886f9  0deb4b6a8  34b15f335  a8cb14b00  2f0771a37  \
0  000137c73        0.0        0.0        0.0        0.0        0.0   
1  00021489f        0.0        0.0        0.0        0.0        0.0   
2  0004d7953        0.0        0.0        0.0        0.0        0.0   
3  00056a333        0.0        0.0        0.0        0.0        0.0   
4  00056d8eb        0.0        0.0        0.0        0.0        0.0   

   30347e683  d08d1fbe3  6ee66e115  20aa07010  ...  3ecc09859  9281abeea  \
0        0.0        0.0        0.0        0.0  ...        0.0        0.0   
1        0.0        0.0        0.0        0.0  ...        0.0        0.0   
2        0.0        0.0        0.0        0.0  ...        0.0        0.0   
3        0.0        0.0        0.0        0.0  ...        0.0        0.0   
4        0.0        0.0        0.0        0.0  ...        0.0        0.0   

   8675bec0b  3a13ed79a  f677d4d13  71b203550  137efaa80  fb36b89d9  \
0        0.0        0.0        0.0        0.0        0.0        0.0   
1        0.0        0.0        0.0        0.0        0.0        0.0   
2        0.0        0.0        0.0        0.0        0.0        0.0   
3        0.0        0.0        0.0        0.0        0.0        0.0   
4        0.0        0.0        0.0        0.0        0.0        0.0   

   7e293fbaf  9fc776466  
0        0.0        0.0  
1        0.0        0.0  
2        0.0        0.0  
3        0.0        0.0  
4        0.0        0.0  

[5 rows x 4992 columns]